cpp/src/IceSSL/OpenSSLEngine.cpp

```diff
@@ -931,20 +931,14 @@ OpenSSL::SSLEngine::destroy()
     if(_ctx)
     {
         SSL_CTX_free(_ctx);
-	_ctx = 0;
+        _ctx = 0;
     }
 }
 
 void
 OpenSSL::SSLEngine::verifyPeer(const string& address, const IceSSL::ConnectionInfoPtr& info, const string& desc)
 {
-#if defined(OPENSSL_VERSION_NUMBER) && OPENSSL_VERSION_NUMBER < 0x10002000L
-    //
-    // Peer hostname verification is new in OpenSSL 1.0.2 for older versions
-    //  we use IceSSL build in hostname verification.
-    //
-    verifyPeerCertName(address, info);
-#endif
+    // keep for binary compatibility.
```

Comment:
Isn't `OpenSSL::SSLEngine` private? I would remove this method, no external code should be calling on this class afaict.